mergify_engine/mergify_pull.py

```diff
@@ -168,6 +169,42 @@ def _get_checks(self):
                                for s in self._get_statuses()])
         return generic_checks
 
+    def _get_team_members(self, name):
+        organization, _, team_slug = name.partition("/")
+        if organization[0] != "@" or not team_slug or '/' in team_slug:
+            # Not a team slug
+            return
+
+        try:
+            g_organization = self.g.get_organization(organization[1:])
+            for team in g_organization.get_teams():
+                if team.slug == team_slug:
+                    return [m.login for m in team.get_members()]
+        except github.GithubException as e:
+            if e.status == 400:
+                return
+            elif e.status >= 500:
```

Comment:
why do you raise if >= 500 ? If you get 5xx it seems like it's a GitHub issue, I'd say we just want to log something, maybe at a the error level.

```python
if e.status != 400:
    self.log.error(…)


return []
```

Reply:
I reraise any 500 from GitHub because we have generic celery task retrying in such case:

https://github.com/Mergifyio/mergify-engine/blob/master/mergify_engine/worker.py#L63